.readthedocs.yaml

```
@@ -0,0 +1,12 @@
+# Read the Docs configuration file
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)